src/include/daos/metrics.h

```diff
@@ -0,0 +1,82 @@
+/**
+ * (C) Copyright 2016-2024 Intel Corporation.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
+ */
+/**
+ * This file is part of daos
+ *
+ * src/include/daos/metrics.h
+ */
+
+#ifndef __DAOS_METRICS_H__
+#define __DAOS_METRICS_H__
+
+#include <daos/common.h>
+#include <daos/tls.h>
+#include <daos_types.h>
+#include <gurt/telemetry_common.h>
+
+/* For now TLS is only enabled if metrics are enabled */
+#define METRIC_DUMP_ENV "DAOS_METRIC_DUMP_ENV"
+#define DAOS_CLIENT_METRICS_ENV	"DAOS_CLIENT_METRICS"
+#define DAOS_CLIENT_METRICS_RETAIN_ENV	"DAOS_CLIENT_METRICS_RETAIN"
```

Comment:
some inconsistency in the names here.
first one should include CLIENT, remove ENV, and missing S in MERICS ->   DAOS_CLIENT_METRICS_DUMP
second one does not include any action, so should be something like -> DAOS_CLIENT_METRICS_ENABLE